site/source/docs/porting/asyncify.rst

```diff
@@ -217,10 +238,46 @@ You can build this with
 This example calls the Promise-returning ``window.crypto.subtle()`` API (the
 example is based off of
 `this MDN example <https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#Basic_example>`_
-). Note how we pass the value to be returned into ``wakeUp()``. We must also
-return the value returned from ``handleSleep()``. The calling C code then
+).
+
+Note that we must propagate the value returned from ``handleSleep()``. The calling C code then
 gets it normally, after the Promise completes.
 
+If you're using ``handleSleep`` API, the value needs to be also passed to the ``wakeUp`` callback, instead of being returned from our handler:
```

Comment:
```suggestion
If you're using the ``handleSleep`` API, the value needs to be also passed to the ``wakeUp`` callback, instead of being returned from our handler:
```